src/coreclr/gc/gcee.cpp

```diff
@@ -451,6 +456,7 @@ segment_handle GCHeap::RegisterFrozenSegment(segment_info *pseginfo)
     heap_segment_used(seg) = heap_segment_allocated(seg);
     heap_segment_plan_allocated(seg) = 0;
 #ifdef USE_REGIONS
+    // TODO - that can be interesting
```

Comment:
I have no idea what this comment is intended to convey, but if it makes sense to GC team it won't matter to me.